internal/js/modules/k6/browser/browser/page_mapping.go

```diff
@@ -749,11 +757,15 @@ func parseWaitForFunctionArgs(
 	return js, popts, exportArgs(gargs), nil
 }
 
-func parseStringOrRegex(v sobek.Value) string {
+func parseStringOrRegex(v sobek.Value, doubleQuote bool) string {
```

Comment:
Could you add a comment to explain use cases for double quote and those for single quote?